### PR TITLE
Feature/43 workflow

### PR DESCRIPTION
**Pull request by ntchou**

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -1,78 +1,19 @@
...
+import HeaderStyled from './Header.styled.js'
+import Logo from '../Logo'
+
+
+const Header = () => (
+    <HeaderStyled>
+        <Container className="header">
+            <Logo />
+            <Container className="header-baseline">
+                <p className="header-baseline-content">Joignez un proche en visio, en un clic, gratuitement.</p>
```
Comment (HugoGresse) [inline src/components/Header/Header.js]: not translated

File: src/components/LegalMentions/LegalMentions.js [hunk 2]
```diff
@@ -1,27 +1,76 @@
...
-    p a, strong {
-        color: ${theme.color.white};
-    }
+import LegalMentionsStyled from './LegalMentions.styled' 
 
-    &-title {
-        font-size: ${theme.font.XL};
-        margin-bottom: ${theme.spacing.M};
-        color: ${theme.color.white};
+const LegalMentions = () => (
```
Comment (HugoGresse) [inline src/components/LegalMentions/LegalMentions.js]: maybe we can put the LegalMentions in markdown to it's easier to read and split them for intertionalization. 
Could be done in another PR though.

File: src/index.js [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
 import theme from './styles/theme'
+import './i18n/i18n'
 
 const rootComponent = (
-    <BrowserRouter>
-        <ThemeProvider theme={theme}>
-            <App/>
-        </ThemeProvider>
-    </BrowserRouter>
+    <Suspense fallback={null}>
```
Comment (HugoGresse) [inline src/index.js]: why using a Suspense here?

Comment (ntchou) [reply src/index.js]: For i18n. Because an error was throwned. I'll try to remove it

**Comment by ntchou** [comment]
Branch is not rebased with master

**Pull request merged**